packages/react-components/src/components/RichTextEditor/Plugins/UpdateContentPlugin.tsx

```diff
@@ -68,6 +68,10 @@ export class UpdateContentPlugin implements EditorPlugin {
         this.onUpdate(UpdateEvent.Dispose);
         break;
 
+      case PluginEventType.CompositionEnd:
+        this.onUpdate(UpdateEvent.ContentChanged);
```

Comment:
Is it ok to always call this, or should there be a check to see if the content has actually changed?

Reply:
it should be okay as the checks will be handled by state hooks and hooks dependencies list in rich text editor and input box.